docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "1.0"
+services:
+  db:
+    container_name: buildingmotif-db
+    image: postgres
+    environment:
+       - POSTGRES_DB=${POSTGRES_DB}
+       - POSTGRES_USER=${POSTGRES_USER}
+       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+    ports:
+      - target: 5432
+        published: 5432
+        protocol: tcp
+        mode: host
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-d", "db_prod"]
```

Comment:
`CMD-SHELL` didn't work for me with [podman-compose](https://github.com/containers/podman-compose/). Did it work with `docker-compose`? Looks like `CMD` takes multiple strings but `CMD-SHELL` only takes one string. Changing it to `CMD` did run all steps successfully for me. https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck

>ValueError: 'CMD_SHELL' takes a single string after it

Reply:
I didn't have any trouble with `docker-compose` but changing to `CMD` seems to better match the docs. So, I would support that change